spec/system/user_page/user_preferences_security_spec.rb

```diff
@@ -88,13 +88,24 @@
       user_menu.sign_out
 
       # login with the key we just created
+      # this triggers the conditional UI for passkeys
+      # which uses the virtual authenticator
       find(".d-header .login-button").click
-      find(".passkey-login-button").click
```

Comment:
This is the flakey fix. The passkey conditional UI seems to automatically invoke the virtual authenticator, and that means that the auth procedure has been simulated before calling `find(".passkey-login-button").click`.